.github/workflows/prod-docker.yml

```diff
@@ -0,0 +1,110 @@
+name: Production Superhero Backend Pipeline
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types: [created]
+
+env:
+  ENV: "prd"
+  APP: "superhero-backend"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    name: Production Pipeline
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - uses: GoogleCloudPlatform/release-please-action@v3
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Use aeternity bot's token else it won't trigger the other gh actions

Reply:
`secrets.BOT_GITHUB_TOKEN`

Reply:
Ye. lets we see tomorrow , where we want to run it now AWS or Hetzner